### ISSUE TITLE
Dummy Output Only on Lenovo Flex 5 (Taeko) in Debian 12 (Bookworm)

### ISSUE THREAD

**Issue by perspector** [WeirdTreeThing/chromebook-linux-audio#204]
First of all, thank you for all of your hard work! It is very appreciated!

**Describe the bug**
I am running the 6.6.58-chrultrabook kernel and ran `setup-audio` and rebooted. However, I still only have "Dummy Output" as my audio device. There should be a headphone jack and speakers.

**Boardname**
Taeko

**Logs**
The log is attached.
[debug-logs-Taeko-2024-12-22_22h02m.tar.gz](https://github.com/user-attachments/files/18224664/debug-logs-Taeko-2024-12-22_22h02m.tar.gz)

Thank you very much!

**Comment by perspector** [comment]
`aplay -l` returns:
```
**** List of PLAYBACK Hardware Devices ****
card 0: sofrt5682 [sof-rt5682], device 0: Speakers (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: sofrt5682 [sof-rt5682], device 1: Headset (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: sofrt5682 [sof-rt5682], device 2: HDMI1 (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: sofrt5682 [sof-rt5682], device 3: HDMI2 (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: sofrt5682 [sof-rt5682], device 4: HDMI3 (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: sofrt5682 [sof-rt5682], device 5: HDMI4 (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0
```
In case that is helpful.

**Comment by perspector** [comment]
I just checked `pavucontrol` (a PulseAudio configuration app), and went to the last tab, `Configuration`. I saw that the sound card was not using any configuration (see screenshot).

![Screenshot from 2024-12-23 07-15-27](https://github.com/user-attachments/assets/01ba08a7-c01c-49e7-b79a-46e15df78430)

So I selected the configuration option and changed it to `Pro Audio`, and **now it works!**

![Screenshot from 2024-12-23 07-14-08](https://github.com/user-attachments/assets/a793b481-f6b2-45fb-a1b8-cb97b3203532)

I am not sure why, but maybe this works for someone else too. Can anyone reproduce the issue and test it? If this is a solution, it could be helpful to add it to the chrultrabook or mrchromebox docs.
